ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -24,7 +24,8 @@ uint64_t getNewTraceIdx() {
 
 #define TRACE_FUNC_PREFIX "__yk_compiled_trace_"
 #define YK_NEW_CONTROL_POINT "__ykrt_control_point"
-#define YK_CONTROL_POINT_ARG_IDX 1
+#define YK_CONTROL_POINT_ARG_VARS_IDX 2
```

Comment:
Might be worth a comment against `YK_CONTROL_POINT_ARG_VARS_IDX`. Something like:

> The argument index of the live variables struct in `__ykrt_control_point`

Or maybe it's obvious? Not sure.

Reply:
I think it's probably obvious enough?